src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/HelperTest.java

```diff
@@ -154,4 +157,26 @@ public void updatePostponedDate() {
                 .getHearingDateCollection().get(0).getValue().getPostponedDate());
     }
 
+    @Test
```

Comment:
It's hard to know what you're testing here. Can you improve the method name to make it more obvious?